patterns/newsletter.php

```diff
@@ -7,15 +8,17 @@
 ?>
 
 <!-- wp:group {"style":{"spacing":{"blockGap":"1.25rem"}},"layout":{"type":"constrained","justifyContent":"left"}} -->
-<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"medium"} -->
-<h2 class="has-medium-font-size" style="text-transform:uppercase"><?php echo esc_html__( 'Newsletter', 'course' ); ?></h2>
-<!-- /wp:heading -->
+<div class="wp-block-group">
+      <!-- wp:heading {"style":{"typography":{"textTransform":"uppercase"}},"fontSize":"medium"} -->
+      <h2 class="has-medium-font-size" style="text-transform:uppercase"><?php echo esc_html__('Newsletter', 'course'); ?></h2>
```

Comment:
Could we preserve the [PHP coding standards for whitespace](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage) in this file? Looks like they were correct before. We'll need to get a linter set up at some point to make catching violations easier.